src/routes/Project/containers/ProjectParamsContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+
+import ProjectParams from '../components/ProjectParams';
+import { updateTempo, updateTimeSig, updateTimeDiv } from '../actions/TableActions';
+
+const mapDispatchToProps = {
+  updateTempo,
+  updateTimeSig,
+  updateTimeDiv
```

Comment:
either way works. we can always change it later if it becomes unwieldy with the sync.